apps/guardian-ui/src/setup/SetupContext.tsx

```diff
@@ -23,19 +23,7 @@ const LOCAL_STORAGE_SETUP_KEY = 'setup-guardian-ui-state';
  * Creates the initial state using loaded state from local storage.
  */
 function makeInitialState(loadFromStorage = true): SetupState {
```

Comment:
Hey @Kodylow , this is the actual fix to the bug! Nice fix. Could we just scope the PR to changes in this file and we land it?